go/src/koding/klient/app/defertime.go

```diff
@@ -0,0 +1,82 @@
+package app
+
+import "time"
+
+// DeferTime allows to invoke registered function after provided duration. It
+// can cancel deferred function call even if internal timer was already started.
+type DeferTime struct {
```

Comment:
oh you are right, I thought that `logFiles` is a channel:d. But you see my point .